docs_user/modules/proc_migrating-ceph-post.adoc

```diff
@@ -0,0 +1,27 @@
+[id="updating-the-cluster-dashboard-configuration_{context}"]
+
+= Updating the {CephCluster} cluster Dashboard configuration
```

Comment:
```suggestion
= Updating the {CephCluster} cluster Ceph Dashboard configuration
```

One nit, because I think "Ceph Dashboard" is the full official name.

Reply:
ack works for me.